src/core/directives/google-map.ts

```diff
@@ -79,6 +79,11 @@ export class SebmGoogleMap implements OnChanges, OnInit {
   zoom: number = 8;
 
   /**
+   * Enables/disables if map is draggable.
+   */
+  draggable: boolean = true;
```

Comment:
ahh right, the dom draggable attribute. If someone wants to allow a user to drag a map around on the screen. Good Catch.
http://www.w3schools.com/tags/att_global_draggable.asp